ui/components/connections/meshSync/Stepper/StepperContent.js

```diff
@@ -248,38 +231,36 @@ export const CredentialDetails = ({ sharedData, handleNext, handleRegistrationCo
       credential.id = selectedCredential?.id;
     }
 
-    dataFetch(
-      '/api/integrations/connections/register',
-      {
-        method: 'POST',
-        credentials: 'include',
-        body: JSON.stringify({
-          skip_credential_verification: skipCredentialVerification,
-          kind: sharedData?.kind, // this is "kind" column of the current row which is selected in the meshsync table. i.e. the entry against which registration process has been invoked.
-          name: sharedData?.componentForm?.name, // This name is from the name field in schema
-          type: sharedData?.connection?.connection?.model?.category?.name?.toLowerCase(),
-          sub_type: sharedData?.connection?.connection?.metadata?.subCategory.toLowerCase(),
-          metadata: sharedData?.componentForm,
-          credential_secret: credential,
-          id: sharedData?.connection?.id,
-          status: 'register',
-        }),
-      },
-      (result) => {
-        if (result === '') {
-          setIsSuccess(true);
-          connectToConnection();
-        } else {
-          setIsSuccess(false);
-        }
-      },
-      (err) => {
-        if (err != '') {
-          console.error(err);
-          setIsSuccess(false);
-        }
-      },
-    );
+    connectionRegister({
+      body: JSON.stringify({
+        skip_credential_verification: skipCredentialVerification,
+        kind: sharedData?.kind, // this is "kind" column of the current row which is selected in the meshsync table. i.e. the entry against which registration process has been invoked.
+        name: sharedData?.componentForm?.name, // This name is from the name field in schema
+        type: sharedData?.connection?.connection?.model?.category?.name?.toLowerCase(),
+        sub_type: sharedData?.connection?.connection?.metadata?.subCategory.toLowerCase(),
+        metadata: sharedData?.componentForm,
+        credential_secret: credential,
+        id: sharedData?.connection?.id,
+        status: 'register',
+      }),
+    })
+      .unwrap()
+      .then(
+        (result) => {
+          if (result === '') {
+            setIsSuccess(true);
+            connectToConnection();
```

Comment:
What does "ConnectToConnection" mean? Could this name be more descriptive, and less redundant?